src/services/programService/index.js

```diff
@@ -18,33 +18,32 @@
  */
 
 /*
- * This file dynamically generates a gRPC client from Ego.proto.
+ * This file dynamically generates an HTTP client from Ego.proto.
  * The content of Ego.proto is copied directly from: https://github.com/icgc-argo/argo-proto/blob/4e2aeda59eb48b7af20b462aef2f04ef5d0d6e7c/ProgramService.proto
  */
 
-import * as grpc from './grpcClient.js';
 import * as http from './httpClient.js';
 
 export default {
 	getPrivateProgram: http.getPrivateProgram,
 	listPrivatePrograms: http.listPrivatePrograms,
 	getJoinProgramInvite: http.getJoinProgramInvite,
-	listUsers: grpc.listUsers,
+	listUsers: http.listUsers,
 
-	listCancers: grpc.listCancers,
-	listPrimarySites: grpc.listPrimarySites,
-	listRegions: grpc.listRegions,
-	listInstitutions: grpc.listInstitutions,
-	listCountries: grpc.listCountries,
+	listCancers: http.listCancers,
+	listPrimarySites: http.listPrimarySites,
+	listInstitutions: http.listInstitutions,
+	listRegions: http.listRegions,
+	listCountries: http.listCountries,
+	listDataCenters: http.listDataCenters,
 
-	createProgram: grpc.createProgram,
-	updateProgram: grpc.updateProgram,
+	createProgram: http.createProgram,
+	updateProgram: http.updateProgram,
 
-	inviteUser: grpc.inviteUser,
-	joinProgram: grpc.joinProgram,
-	updateUser: grpc.updateUser,
-	removeUser: grpc.removeUser,
+	inviteUser: http.inviteUser,
+	joinProgram: http.joinProgram,
+	updateUser: http.updateUser,
+	removeUser: http.removeUser,
 
 	getPublicProgram: http.getPublicProgram,
-	listDataCenters: http.listDataCenters,
 };
```

Comment:
I don't understand this file. Isn't this just the same as exporting the http object `export default http`?
And even then that's all it's doing. There's no mapping to new property names. Just a re-export.

Reply:
I was questioning this as well, I will investigate

Reply:
Agree we can simplify this now. Previously some endpoints were grpc and some were http. Now this isn't needed.

Reply:
Changed `httpClient` to just `index` and added exports to the file
https://github.com/icgc-argo/platform-api/pull/714/commits/3de29a1768d66dfe6cce44a58673192e757f30e6